simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/IdentityVariableAccessor.java

```diff
@@ -51,6 +55,14 @@ public String getVariableName() {
         return variableValidators;
     }
 
+    @Override
+    public List<ReportDatum> generateReportData(String componentName) {
```

Comment:
Method argument is not used

Reply:
Well, It's an overriden method from an abstract class, I don't know what do you want me to do here?

Reply:
remove this method from here... and dont use abstract keyword for this method. Just declare it 
 in the skeleton class.